tests/components/automation/test_time.py

```diff
@@ -53,6 +53,37 @@ async def test_if_fires_using_at(hass, calls):
     assert calls[0].data["some"] == "time - 5"
 
 
+async def test_if_fires_using_multiple_at(hass, calls):
+    """Test for firing at."""
+    assert await async_setup_component(
```

Comment:
We probably need to patch time like we do in `tests/components/automation/test_time_pattern.py` since the event loop still ticks forward.

If the test is setup at 4:59:59.999999 it will probably fire twice.

Reply:
Done